core/src/test/java/org/elasticsearch/index/query/SimpleIndexQueryParserTests.java

```diff
@@ -1098,7 +1098,6 @@ public void testTermsQueryOptions() throws IOException {
 
         assertTrue("terms query disable_coord option mismatch", booleanQuery.isCoordDisabled());
         assertThat(booleanQuery.getBoost(), equalTo(2.0f));
-        assertThat(booleanQuery.getMinimumNumberShouldMatch(), equalTo(2));
```

Comment:
The option is deprecated, and now no longer supported, unless we want to still keep it.